client/features/sharedComponents/MediaTileCarousel.jsx

```diff
@@ -0,0 +1,166 @@
+import React from 'react';
+import TempMediaTile from './TempMediaTile.jsx';
+import './MediaTileCarousel.css';
+
+class MediaTileCarousel extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.handleScroll = this.handleScroll.bind(this);
+    this.handleAction = this.handleAction.bind(this);
+    this.handleNav = this.handleNav.bind(this);
+    this.handleResize = this.handleResize.bind(this);
+    this.containerElement = React.createRef();
+
+    window.addEventListener('resize', this.handleResize);
+
+    this.containerWidth = 0;
+    this.tileWidth = 0;
+    this.maxScroll = 0;
+    this.navLeftSymbol = String.fromCharCode(12296);
+    this.navRightSymbol = String.fromCharCode(12297);
+
+    this.state = {
+      showLeftNav: false,
+      showRighttNav: false
```

Comment:
typo: showRightNav has two t's here.

Reply:
Wow. Thank you. I could't figure out why that arrow didn't appear when it was supposed to.